WEP0000_wep-proposal.rst

```diff
@@ -0,0 +1,228 @@
+- **Allow edits from maintainers** option is selected. This helps the reviewer merge your WEP draft if it is accepted.
+- The PR is tagged with a **draft** flag, to indicate that it is in a draft state and is ready for pre-review. 
+
+
+After the pull request for the WEP draft is submitted, a WEP team member reviews the pull request to make sure it adheres to the guidelines. If the WEP is clearly not ready, the reviewer might reject the pull request and ask the author to submit a new WEP after the problems have been fixed.
+
+The reviewer does not vote on the proposed change itself, and only reviews the WEP for scope compliance, format, and completeness. The pre-review period is usually one week, but might change based on the scope of the pre-review feedback and the availability of the author to implement changes.
+
+After the WEP is ready for the community review, the reviewer assigns the **ready-for-discussion** label to the PR. The CoreOps team is then responsible for distributing announcements of each WEP to the relevant community segment. For example, a WEP about meetups will be shared with the Meetup team for distribution between the meetup organizers. For some WEPs, community-wide announcements will be made on Slack or to the mailing list. 
+  
+Community review, discussion, and updates
+.........................................
+
+The standard community review period for a WEP is **30 days**. The WEP is reviewed by a process called **lazy consensus**. This means that community members either provide feedback, approve the WEP explicitly, or ignore the WEP. 
+
+If the review period ends without a clear consensus, but further discussion is still needed, the review period might be extended by the WEP team. The review period is never shorter than one month, but other WEPs can be submitted in the meantime, and multiple WEPs might be undergoing a community review simultaneously. 
+
+During the community review period, the WEP goes through one or more iterations of feedback and updates. Community feedback is submitted in the form of comments to the PR, and the WEP author is responsible to update the WEP content in case the incoming feedback is accepted.
+
+All discussion on the WEP must happen on the PR in the form of comments in order to be considered as official feedback on the WEP. Of course external conversations can happen on other platforms, but the official record for a WEP discussion is only on the PR itself. If discussion happens on another platform, we recommend copying the most important points or the results of those discussions into the WEP comments.
+
+You must have a GitHub account to post comments to a PR. If you need help getting started with GitHub, email support@writethedocs and the WTD team will help you set up your account and get familiar with GitHub basics. 
```

Comment:
Should this be `support@writethedocs.org`?

Reply:
It should be, thanks!